cmd/containerd/main.go

```diff
@@ -104,35 +108,35 @@ high performance container runtime
 		}
 
 		// Get events publisher
-		nec, err := getNATSPublisher(context)
+		nec, err := getNATSPublisher(ea)
 		if err != nil {
 			return err
 		}
 		defer nec.Close()
-		ctx := log.WithModule(gocontext.Background(), "containerd")
-		ctx = log.WithModule(ctx, "execution")
-		ctx = events.WithPoster(ctx, events.GetNATSPoster(nec))
 
 		var (
 			executor execution.Executor
 			runtime  = context.GlobalString("runtime")
 		)
+		log.G(ctx).WithField("runtime", runtime).Info("run with runtime executor")
+		ctx2 := log.WithModule(ctx, "execution")
```

Comment:
I separated it as @mlaventure suggested, and I think it reasonable because otherwise we'll see some containerd messages with model `containerd/execution/shim`.